deploy/charts/templates/deployment.yaml

```diff
@@ -29,8 +29,8 @@ spec:
       containers:
       - name: {{ .Chart.Name }}
         image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}" # In Jenkins, need to call helm with --set image.tag=master-XY
-        command: ["npm"]
-        args: ["start", "--", "--port={{ .Values.service.port }}", "--config={{ .Values.deployenv.environment }}", "--prefix", "/ui"]
+        command: ["tini"]
+        args: ["--", "npm", "start", "--", "--port={{ .Values.service.port }}", "--config={{ .Values.deployenv.environment }}"]
```

Comment:
K8s calls these when starting the container. `command` here maps to and overrides `ENTRYPOINT` in Dockerfile.jenkins. Likewise, `args` here overrides `CMD` in the Dockerfile.